wasp-app-runner/src/waspCli.ts

```diff
@@ -0,0 +1,121 @@
+import { stripVTControlCharacters } from "node:util";
+
+import { log } from "./logging.js";
+import { getWaspcDirAbsPath } from "./path.js";
+import { processManager } from "./process.js";
+import { DbType } from "./db/index.js";
+
+export function migrateDb({
+  waspCliCmd,
+  pathToApp,
+  extraEnv,
+}: {
+  waspCliCmd: string;
+  pathToApp: string;
+  extraEnv: Record<string, string>;
+}): Promise<number | null> {
+  return processManager.spawnWithLog({
+    name: "migrate-db",
+    cmd: waspCliCmd,
+    args: ["db", "migrate-dev"],
+    cwd: pathToApp,
+    extraEnv,
+  });
+}
+
+export function startApp({
+  waspCliCmd,
+  pathToApp,
+  extraEnv,
+}: {
+  waspCliCmd: string;
+  pathToApp: string;
+  extraEnv: Record<string, string>;
```

Comment:
I recommend using for this type since it's mentioned so often. I'd also go for an index signature to name the key:
```ts
type EnvVars = { [envVarName: string]: string }
```